### PR TITLE
Add linux-i2cdev to e-h::i2c::ErrorKind mapping

### PR DESCRIPTION
**Pull request by ryankurte**

### CODE REVIEW

File: src/i2c.rs [hunk 1]
```diff
@@ -138,11 +147,26 @@ impl From<i2cdev::linux::LinuxI2CError> for I2CError {
...
+            i2cdev::linux::LinuxI2CError::Nix(e) => *e,
+            i2cdev::linux::LinuxI2CError::Io(e) => {
+                match e.raw_os_error() {
+                    Some(r) => nix::Error::from_i32(r),
+                    None => return ErrorKind::Other,
+                }
+            },
+        };
+
+        // https://01.org/linuxgraphics/gfx-docs/drm/i2c/fault-codes.html
```
Comment (posborne) [inline src/i2c.rs:163]: This is probably the preferred link : https://www.kernel.org/doc/html/latest/i2c/fault-codes.html

**Comment by posborne** [comment]
The mapping here looks reasonable to me.

**Review by posborne — APPROVED** [review]

**Review by eldruin — DISMISSED** [review]
Looks good to me as well, thanks!
bors r+

**Comment by ryankurte** [comment]
there's a `Cargo.toml` error buried in there somewhere... just rebasing

**Review by eldruin — DISMISSED** [review]
Thanks!
bors r+

**Review by eldruin — APPROVED** [review]
Third time's the charm.
bors r+

**Pull request merged**